app/controllers/host/vms_controller.rb

```diff
@@ -0,0 +1,28 @@
+class Host::VmsController < ApplicationController
```

Comment:
I'm fine with that - `import` sounds more meaningful to me though. I played with that `new` route a while ago to write a plugin for more OpenStack stuff, it might be possible that plugins define their own routes to it.